src/tests/suite/daos_rebuild.c

```diff
@@ -1483,7 +1483,7 @@ rebuild_delay_and_reintegrate(void **state)
 	if (!test_runable(arg, 6))
 		return;
 
-	rc = daos_pool_set_prop(arg->pool.pool_uuid, "self_heal", "delay_rebuild");
```

Comment:
`rebuild_delay_and_reintegrate` is functional test `test_rebuild_33`, which is a daily test and not ran on this PR.